get_tidy_conf.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# Copy the tidy.conf file from an ebookmaker version into the dist directory
+# Note that ebookmaker versions are normally tagged not released
+# The tidy.conf file is not expected to change often
+
+set -e
+
+VERSION=0.11.26
+
+rm -rf ebookmaker.zip ebmtemp
+curl -L -o ebookmaker.zip https://github.com/gutenbergtools/ebookmaker/archive/refs/tags/$VERSION.zip
```

Comment:
And if we wanted to be extra fancy we could have this script auto-detect the version from `Pipfile` so that didn't have to be updated:
```bash
VERSION=$(grep ebookmaker Pipfile | tr -d '"' | sed 's/.*==//')
```
That's assuming the Windows *nix env includes `tr` and `sed`.